coforma/swift-tech-challenge
# add dynamodb lib

# Summary

Since our data goes through some processing before storing it in the database we will use the AWS database for local development. This saves time in ensuring both local and deployed instances go through a similar seeding process.

- Adds a utility/ folder in our repo with tools for interacting with dynamodb, locally and deployed
- Adds necessary packages
- Adds documentation





Fixes #23 

## Type of change

Please delete options that are not relevant.


# How To Test
Temporarily modify your PageHeader as follows
```js
import dynamoClient from "../../utils/dynamodb-lib";

export const PageHeader = async () => {
  const params = {
    TableName: "institutions",
    Key: { institutionId: 110608, recordType: "data" },
  };
  const response = await dynamoClient.get(params);
  console.log("response", response);
...
```

- Run `yarn` to install the latest dependencies
- Run `yarn dev`
- Once the app loads, verify your terminal (where you ran `yarn dev`) shows the output of the item you created

# Checklist:
